client/src/routes/WorkspaceRoute/Workspace.css

```diff
@@ -39,7 +39,9 @@ body {
   border-bottom: 6px solid #02feff;
 }
 
-.wrapper__column div {
+.wrapper__column .column_samples,

```

Comment:
I think when naming `column_samples` & `column_soundeffects`, I forgot to write two underscores. Could you please change these two class names to `column__samples` and `column__sound-effects`?

Reply:
Done! 👍🏽 (I used `column__soundeffects` instead of `column__sound-effects` since they're both the same thing and the first is easier to type).